app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,14 @@
   <link rel="icon" type="image/x-icon" href="/favicon.ico">
   <link rel="stylesheet" type="text/css" href="<%= theme_stylesheet_url %>">
 
+  <% codemirror_assets_css.each do |it| %>
+      <link rel="stylesheet" type="text/css" href="<%= it %>">
+  <% end %>
+
+  <% codemirror_assets_js.each do |it| %>
```

Comment:
I think that if we place those assets on vendor folder, this is not necessary. 

Moreover, I am not sure assets included this way will be minified :scream: